benches/parser.rs

```diff
@@ -25,37 +25,36 @@ fn load_example(example: &str) -> Vec<String> {
         .collect()
 }
 
-// Register a `fibonacci` function and benchmark it over multiple cases.
 #[divan::bench(args = ["nix-shell.rec", "nixos-rebuild.rec"])]
 fn parse_raw(bencher: Bencher, example: &str) {
-    let example = load_example(example);
+    let lines = load_example(example);
 
     bencher
-        .with_inputs(|| &example)
-        .counter(divan::counter::ItemsCount::new(example.len()))
-        .bench_refs(|lines| {
+        .counter(lines.len())
+        .bench(|| {
             lines
                 .iter()
                 .inspect(|line| {
-                    let _: RawAction = serde_json::from_str(line).expect("invalid line");
+                    let action: RawAction = serde_json::from_str(line).expect("invalid line");
+                    divan::black_box_drop(action);
                 })
                 .count()
```

Comment:
Instead of `.inspect(...).count()` it might make sense for the benchmark use a `for` loop or `.for_each(...)`.